lib/harness/active_publisher.rb

```diff
@@ -48,6 +50,12 @@ module ActivePublisher
       ::Harness.timing LATENCY_METRIC, event.duration
     end
 
+    ::ActiveSupport::Notifications.subscribe "publishes_confirmed.active_publisher" do |*args|
+      event = ::ActiveSupport::Notifications::Event.new(*args)
+      ::Harness.count PUBLISH_CONFIRMED_METRIC, 1
```

Comment:
Should this be count or increment?

Reply:
We do publish multiple messages to an exchange before waiting for the publishes to be confirmed, but I think this is fine to incr one by one for now.

Reply:
It should be increment good call.  Also in terms of publishing multiple messages to an exchange, that sounds like the corresponding MR in active publisher for the active support notification that handles this concern (ie. active support is notified in active publisher accordingly only when it attempts to wait for confirms)